README.md

```diff
@@ -199,6 +187,7 @@ This allows for a local model name that is different from the remote resource na
 
 For example:
 
+// @TODO: How to do that in LB4?
```

Comment:
Was not able to upgrade from LB3 to LB4 even checking the current code/documentation/issues/....